jenkins-pipelines/nemesis_custom.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
```

Comment:
to test nemesis you can use SCT byo job. no need to create a Jenkins file

Reply:
done